### PR TITLE
react-876-evaluation-admin-reopen

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Evaluation Admin Reopen
[#876](https://github.com/nerubia/kh360-react/issues/876)
BE:https://github.com/nerubia/kh360-nodejs/pull/513

### CODE REVIEW

File: src/redux/slices/evaluation-administration-slice.ts [hunk 1]
```diff
@@ -140,9 +140,11 @@ export const publishEvaluationAdministration = createAsyncThunk(
 
 export const reopenEvaluationAdministration = createAsyncThunk(
   "evaluationAdministration/reopen",
-  async (id: number, thunkApi) => {
+  async ({ id, endDate }: { id: number | string; endDate: Date | string }, thunkApi) => {
```
Comment (emmaacha) [inline src/redux/slices/evaluation-administration-slice.ts:143]: Put this as an interface (we can call this ReopenData) inside `evaluation-admnistration-type`

Comment (emmaacha) [reply src/redux/slices/evaluation-administration-slice.ts:143]: You can check how it is done for `sendReminder`

**Review by emmaacha — CHANGES_REQUESTED** [review]
There's also a bug where if I input nothing on the date and click reopen, this error occurs: `You can't reschedule earlier than today.` Kindly check, thanks!

File: src/redux/slices/evaluation-administration-slice.ts (hunk 1 above)
Comment (chadiegilnerubia) [reply src/redux/slices/evaluation-administration-slice.ts]: done :3

**Review by emmaacha — APPROVED** [review]

**Pull request merged**